src/Fan/Helpers/RegexUtilities.cs

```diff
@@ -18,15 +22,14 @@ public static bool IsValidEmail(this string strIn)
             {
                 strIn = Regex.Replace(strIn, @"(@)(.+)$", DomainMapper,
                                       RegexOptions.None, TimeSpan.FromMilliseconds(200));
+                if(string.IsNullOrEmpty(strIn))
+                    return false;
             }
-            catch (RegexMatchTimeoutException)
+            catch (Exception ex)
             {
-                return false;
+                if(ex is RegexMatchTimeoutException || ex is ArgumentNullException)
```

Comment:
If you are checking for null on line 25, why would you check for `ArgumentNullException` on this line?

Reply:
Oh, I forgot to remove it. My mistake!